pages/profile/[userId].tsx

```diff
@@ -230,6 +233,15 @@ export default function InternalProfile({
         <AchievementComponent userId={userId} />
       ),
     },
+    {
+      shouldShow:  userId == "R7nzMKiRewgJuLm54eQ1KdHV3g82",
```

Comment:
@vithushan19  what is the best form when referencing my userId? Or is this fine how it is.

Reply:
At the top of this profile component you should create a const interviewData = getInterviewData(userId)

getInterviewData should be a function defined in the api folder probably in studentPortal/interviews/

In the api function you can do the check against your specific userID and then return your hardcoded data if the id matches. 

If the id doesn't match then return null. 

This way shouldShow can check if the interviewData is null or not.